Exercises/Modules/Chapter 12/Soln12_08/Soln12_08.cpp

```diff
@@ -0,0 +1,101 @@
+// Remove Boxes from a Truckload without searching a second time
```

Comment:
There is indeed something wrong with the friend declarations.
error C2248: 'Truckload::Iterator::Iterator': cannot access private member declared in class 'Truckload::Iterator'

Reply:
bug in VC++?

Reply:
I'd say a bug, yes. Have it with all my Truckloads from this point on, but only in the modular version.